src/main/definitions/case.ts

```diff
@@ -12,6 +12,16 @@ export interface CaseDate {
   day: string;
 }
 
+export interface Address {
+  AddressLine1: string;
+  AddressLine2: string;
+  AddressLine3: string;
```

Comment:
Where does this get used and do we need AddressLine3? I'd also probably have this interface in a new file